libutils/string_lib.h

```diff
@@ -102,6 +102,16 @@ static inline bool CharIsPrintableAscii(const char c)
 }
 bool EmptyString(const char *s);
 
+static inline bool StringContains(const char *const a, const char *const b)
```

Comment:
`needle` and `hay` would be better names for the parameters.

Reply:
Done.